android/library/src/main/java/com/liferay/mobile/screens/asset/display/AssetDisplayScreenlet.java

```diff
@@ -278,18 +278,34 @@ public void setPortletItemName(String portletItemName) {
 		this.portletItemName = portletItemName;
 	}
 
+	public AssetDisplayListener getListener() {
+		return listener;
+	}
+
 	public void setListener(AssetDisplayListener listener) {
 		this.listener = listener;
 	}
 
+	public AssetDisplayInnerScreenletListener getConfigureListener() {
+		return configureListener;
```

Comment:
configuration? other name?

Reply:
The listener was created to configure the "child" (inner) screenlet or render a custom view. So, can we call it... innerScreenletListener? configurationListener?